lms/lib/courseware_search/lms_result_processor.py

```diff
@@ -8,18 +8,16 @@
 from opaque_keys.edx.locations import SlashSeparatedCourseKey
 from search.result_processor import SearchResultProcessor
 from xmodule.modulestore.django import modulestore
-
-from courseware.access import has_access
+from lms.djangoapps.course_blocks.api import get_course_blocks
+from lms.djangoapps.courseware.access import has_access
 
 
 class LmsSearchResultProcessor(SearchResultProcessor):
-
     """ SearchResultProcessor for LMS Search """
     _course_key = None
-    _course_name = None
     _usage_key = None
     _module_store = None
-    _module_temp_dictionary = {}
+    _course_blocks = {}
 
     def get_course_key(self):
         """ fetch course key object from string representation - retain result for subsequent uses """
```

Comment:
I'm not completely sure here but I think this is the new way of parsing course IDs:
https://github.com/edx/edx-platform/wiki/Opaque-Keys-(Locators)